src/signals/incident-management/containers/ReporterContainer/components/FeedbackStatus.tsx

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2021 Gemeente Amsterdam
+import { themeColor } from '@amsterdam/asc-ui'
+import React, { useMemo } from 'react'
+import styled from 'styled-components'
+import { Theme } from 'types/theme'
+import { Feedback } from '../types'
+
+interface FeedbackStatusProps {
+  feedback: Feedback | null
+  className?: string
+}
+
+const Status = styled.span<{ feedback: Feedback | null; theme: Theme }>`
+  ${({ feedback, theme }) => {
+    if (feedback === null || !feedback.submitted_at) return
```

Comment:
We avoid as much as possible using snake casing in the code. Converting the api results to own defined structures is preffered. In this case parameter like 
`isSatisfied? boolean` would be enough and would make this component reusable in the `detail incident` story